### DISCUSSION TITLE
How to plot MRI using pyqtgraph?

### DISCUSSION THREAD

**Discussion by zhengliuX** [pyqtgraph/pyqtgraph#2115] [Q&A]
Hi, I saw an MRI example using pyqtgraph from the example
![image](https://user-images.githubusercontent.com/65115609/142722845-eecdad75-8190-435b-98a0-db5805cd5cf8.png)
May I ask if this code is available? I'm new and I don't know where to start then reproduce this?
Thanks!

**Comment by pijyoi** [comment]
I was able to download some sample DICOM files with a time-axis from http://www.rubomedical.com/dicom_files/
```python
import pyqtgraph as pg
import pydicom

pg.setConfigOptions(imageAxisOrder='row-major')

path = pydicom.data.get_testdata_file("CT_small.dcm")    # included with pydicom
# path = '/work/images/0012.DCM'      # some other file on your filesystem 
ds = pydicom.dcmread(path)

pg.mkQApp()
imv = pg.ImageView()
imv.show()
imv.setImage(ds.pixel_array)

# Start up with an ROI
imv.ui.roiBtn.setChecked(True)
imv.roiClicked()

pg.exec()

```

**Reply by zhengliuX** [reply]
Sorry for the delay!
Thank you so much for the help!
But I was wondering how to view the different slices in three directions at the same time(sagittal, coronal, axial) by scrolling the mouse.

**Reply by pijyoi** [reply]
Could you post the code (and sample data) that you have done so far to achieve your objective?
